python/cudf/cudf/core/buffer/spillable_buffer.py

```diff
@@ -170,11 +171,7 @@ def _from_host_memory(cls, data: Any) -> Self:
         data = data.cast("B")  # Make sure itemsize==1
 
         # Create an already spilled buffer
-        ret = cls.__new__(cls)
-        ret._owner = None
-        ret._ptr = 0
-        ret._size = data.nbytes
-        ret._exposed = False
+        ret = cls(ptr=0, size=data.nbytes, owner=None, exposed=False)
```

Comment:
I see you're being explicit about parameters at the call site. You could use keyword-only arguments in the constructor if you wish, like `def __init__(self, *, ...):`. That would force all call sites to use the names of the arguments.

Reply:
good point, done